src/main/java/hudson/plugins/analysis/collector/AnalysisPublisher.java

```diff
@@ -149,6 +68,21 @@ public boolean isCheckStyleActivated() {
     }
 
     /**
+     * Needed by workflow snippet generator.
+     * Having this ugly getter naming to keep backward configuration file compatibility on fields names.
+     * 
+     * @see #isCheckStyleActivated()
+     */
+    public boolean isIsCheckStyleActivated() {
```

Comment:
No, it doesn't. Anyway the correct naming is `isIsCheckStyleActivated` since the field name is `isCheckStyleActivated`. Perhaps we should change the field name, but it's out of scope in this PR.